concepts/80 Troubleshooting/30 3rd-Party Frameworks Issues/00 Bootstrap/05 Modals.md

```diff
@@ -0,0 +1,68 @@
+If you place DevExtreme components in a [Bootstrap modal](https://getbootstrap.com/docs/5.3/components/modal/), the following issues may occur:
+
+- Users cannot insert, update, or delete a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) row.
+- Users cannot enter values in the [FilterBuilder](/Documentation/Guide/UI_Components/FilterBuilder/Overview/).
+- Editors in the [DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/) lose focus.
+- [TextArea](/Documentation/Guide/UI_Components/TextArea/Getting_Started_with_TextArea/) loses focus.
+- DataGrid [ColumnChooser](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columnChooser/) loses focus.
+
+On page load, Bootstrap locates all modals and places their children into special collections. A modal only allows items from its children collection to receive focus. At the same time, DevExtreme components can generate elements on the fly. Such dynamically-generated elements do not get attached to the focusable collection.
```

Comment:
`their children`? Why do you refer to them as 'children'?

Reply:
Well, the term "child object" comes from [inheritance](https://en.wikipedia.org/wiki/Inheritance_(object-oriented_programming)#:~:text=through%20inheritance%2C%20a%20%22-,child%20object,-%22%2C%20acquires%20all%20the). In this context, "children" are the inner elements of the modals. That means they become influenced by a modal.
I can change "children" to "inner elements" if that's more comprehensible.